uint32 simulation_step
uint64 time
uint64 deadline
string id
Entity[] entities
Dispenser[] dispensers
Block[] blocks
Goal[] goals
Obstacle[] obstacles
Task[] tasks
Agent agent
Team team

